CRM/Exthours/Kimai/Utils.php

```diff
@@ -339,8 +346,16 @@ public static function saveActivityCustomFields($entityId, $customFields, $actio
       "custom_{$workCategory['id']}" => $customFields['work_category'],
     ]);
 
-    // Add empty space string if trackingNumberVal is empty
-    $trackingNumberVal = !empty($customFields['tracking_number']) ? $customFields['tracking_number'] : ' ';
+    // Init trackingNumberValue and IsInvoicedValue
+    $trackingNumberVal = ' ';
```

Comment:
I think this line will cause conflicts now that we've merged pr #6 @lucky091588 . Probably need to rebase the PR.